.travis.yml

```diff
@@ -1,59 +1,56 @@
-branches:
```

Comment:
Do we need to change all travis config on master?

Reply:
With limited compatibility of the v3, maybe we can change the travis version yes.
About coverall, I can add it again. and about travis ```memory_limit = -1``` config, it's maybe better to avoid it to help detect possible memory leak. Used it in the past, no strong opinion on it.